src/common/rollbar-api.js

```diff
@@ -36,18 +35,33 @@ class RollbarAPI {
     return this.processResponse(resp);
   }
 
+  async sigendURLsourcemaps(request) {
+
+    const json = JSON.stringify({ version: request.version , prefix_url: request.baseUrl});
```

Comment:
Axios can accept the javascript object directly, and when doing so, will automatically set the content type. The `JSON.stringify` and `headers` can be removed, and these lines can be simplified to just:
```
const resp = await this.axios.post(
  '/signed_url/sourcemaps',
  { version: request.version , prefix_url: request.baseUrl}
);
```